### ISSUE TITLE
Align license metadata across Adobe dependencies

### ISSUE THREAD

**Issue by G00fY2** [adobe/aepsdk-core-android#546]
We use tools to verify our allowed licenses. Adobe is currently causing issues because of inconsistent license metadata in the maven artifacts:

### GOOD
> "identifier": "Apache-2.0"
> "name": "Apache License 2.0"
> "url": "https://www.apache.org/licenses/LICENSE-2.0"

- com.adobe.marketing.mobile:analytics
- com.adobe.marketing.mobile:assurance
- com.adobe.marketing.mobile:core
- com.adobe.marketing.mobile:identity
- com.adobe.marketing.mobile:lifecycle
- com.adobe.marketing.mobile:sdk-bom
- com.adobe.marketing.mobile:userprofile

### BAD
> "name": "The Apache License, Version 2.0",
> "url": null

- com.adobe.marketing.mobile:audience
- com.adobe.marketing.mobile:signal
- com.adobe.marketing.mobile:target

Full dump of our dependencies. I have not checked all available adobe maven repositories, but just the ones we use in our project.
```json
{
        "groupId": "com.adobe.marketing.mobile",
        "artifactId": "analytics",
        "version": "2.0.3",
        "name": "AdobeMobileAnalyticsSdk",
        "spdxLicenses": [
            {
                "identifier": "Apache-2.0",
                "name": "Apache License 2.0",
                "url": "https://www.apache.org/licenses/LICENSE-2.0"
            }
        ],
        "scm": {
            "url": "https://github.com/adobe/aepsdk-analytics-android"
        }
    },
    {
        "groupId": "com.adobe.marketing.mobile",
        "artifactId": "assurance",
        "version": "2.1.1",
        "name": "AdobeMobileAssurance",
        "spdxLicenses": [
            {
                "identifier": "Apache-2.0",
                "name": "Apache License 2.0",
                "url": "https://www.apache.org/licenses/LICENSE-2.0"
            }
        ],
        "scm": {
            "url": "https://github.com/adobe/aepsdk-assurance-android"
        }
    },
    {
        "groupId": "com.adobe.marketing.mobile",
        "artifactId": "audience",
        "version": "2.0.0",
        "name": "AdobeMobileAudienceSdk",
        "unknownLicenses": [
            {
                "name": "The Apache License, Version 2.0",
                "url": null
            }
        ],
        "scm": {
            "url": "https://github.com/adobe/aepsdk-audience-android"
        }
    },
    {
        "groupId": "com.adobe.marketing.mobile",
        "artifactId": "core",
        "version": "2.4.0",
        "name": "AdobeMobileCoreSdk",
        "spdxLicenses": [
            {
                "identifier": "Apache-2.0",
                "name": "Apache License 2.0",
                "url": "https://www.apache.org/licenses/LICENSE-2.0"
            }
        ],
        "scm": {
            "url": "https://github.com/adobe/aepsdk-core-android"
        }
    },
    {
        "groupId": "com.adobe.marketing.mobile",
        "artifactId": "identity",
        "version": "2.0.3",
        "name": "AdobeMobileCoreSdk",
        "spdxLicenses": [
            {
                "identifier": "Apache-2.0",
                "name": "Apache License 2.0",
                "url": "https://www.apache.org/licenses/LICENSE-2.0"
            }
        ],
        "scm": {
            "url": "https://github.com/adobe/aepsdk-core-android"
        }
    },
    {
        "groupId": "com.adobe.marketing.mobile",
        "artifactId": "lifecycle",
        "version": "2.0.4",
        "name": "AdobeMobileLifecycleSdk",
        "spdxLicenses": [
            {
                "identifier": "Apache-2.0",
                "name": "Apache License 2.0",
                "url": "https://www.apache.org/licenses/LICENSE-2.0"
            }
        ],
        "scm": {
            "url": "https://github.com/adobe/aepsdk-core-android"
        }
    },
    {
        "groupId": "com.adobe.marketing.mobile",
        "artifactId": "sdk-bom",
        "version": "2.3.0",
        "name": "AdobeMobileBOM",
        "spdxLicenses": [
            {
                "identifier": "Apache-2.0",
                "name": "Apache License 2.0",
                "url": "https://www.apache.org/licenses/LICENSE-2.0"
            }
        ],
        "scm": {
            "url": "https://github.com/adobe/aepsdk-core-android"
        }
    },
    {
        "groupId": "com.adobe.marketing.mobile",
        "artifactId": "signal",
        "version": "2.0.1",
        "name": "AdobeMobileCoreSdk",
        "unknownLicenses": [
            {
                "name": "The Apache License, Version 2.0",
                "url": null
            }
        ],
        "scm": {
            "url": "https://github.com/adobe/aepsdk-core-android"
        }
    },
    {
        "groupId": "com.adobe.marketing.mobile",
        "artifactId": "target",
        "version": "2.0.1",
        "name": "AdobeMobileTargetSdk",
        "unknownLicenses": [
            {
                "name": "The Apache License, Version 2.0",
                "url": null
            }
        ],
        "scm": {
            "url": "https://github.com/adobe/aepsdk-target-android"
        }
    },
    {
        "groupId": "com.adobe.marketing.mobile",
        "artifactId": "userprofile",
        "version": "2.0.1",
        "name": "AdobeMobileUserProfileSdk",
        "spdxLicenses": [
            {
                "identifier": "Apache-2.0",
                "name": "Apache License 2.0",
                "url": "https://www.apache.org/licenses/LICENSE-2.0"
            }
        ],
        "scm": {
            "url": "https://github.com/adobe/aepsdk-userprofile-android"
        }
    },
```

**Comment by praveek** [comment]
@G00fY2 I've made sure to fix the license section in all our repositories. You'll be able to see it when the next release for each extension goes live.

**Comment by G00fY2** [comment]
@praveek @yangyansong-adbe I can confirm that this has been fixed with the 3.0 releases (we updated the BoM). The license data in the POMs are now valid and aligned.
